dagger-core/src/main/java/com/gotocompany/dagger/core/processors/external/grpc/client/GrpcClient.java

```diff
@@ -89,6 +99,9 @@ private ClientCall<DynamicMessage, DynamicMessage> createCall(CallOptions callOp
      * Close channel.
      */
     public void close() {
+        if (decoratedChannel != null && decoratedChannel.isShutdown()) {
```

Comment:
`decoratedChannel.isShutdown()` I think this should be negated ?

Reply:
Yes, you are right, made changes.